lexicons/app/bsky/feed/defs.json

```diff
@@ -177,6 +186,81 @@
           "items": { "type": "ref", "ref": "app.bsky.graph.defs#listViewBasic" }
         }
       }
+    },
+    "interaction": {
+      "type": "object",
+      "properties": {
+        "item": { "type": "string", "format": "at-uri" },
```

Comment:
Will this generally be either a post or repost URI?

Reply:
Right now I was thinking it would always be the post uri & any repost that got it in there would be associated through the `feedContext` that gets sent around 🤔

I left it somewhat vague because I think it's likely we'll start passing down non-post objects in feeds at some point